tests/test_iframe.py

```
@@ -0,0 +1,38 @@
+def test_demo_serves_with_templates():
+    # Exercises the compiled frontend templates end to end.
+    example = iFrame().example_inputs()
+    with gr.Blocks() as demo:
+        iFrame(label="Blank")
+        iFrame(value=example, label="Populated")
+
+    _, local_url, _ = demo.launch(prevent_thread_lock=True, show_error=True)
+    try:
+        import urllib.request
+
+        html = urllib.request.urlopen(local_url).read().decode()
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)